### PR TITLE
Added Security topic

### PR DESCRIPTION
**Pull request by shams-ahmed**
would be great to get feedback or ways to improve this area!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -467,6 +467,25 @@ First you should make your app send crash logs onto a server somewhere so that y
...
+
+##### Do's 
+* Store sensitive data in Keychain. if your uncomfortable implementing Keychain library use a wrapper like [UICKeyChainStore](https://github.com/kishikawakatsumi/UICKeyChainStore)
+* Treat untrusted files and data with care, always use NSFileProtectionComplete
+* Remove personal data from memory when it not needed (i.e. self.password = nil;)
+* Protect data in transit (https over ssl)
+* Use TextFields with Secure Option and disable Autocorrection 
+* Clear the pasteboard once the app enters in background
+* UIWebView: set cache policy to ignore local as its stored in the bundle
+* NSLog disable it for release
```
Comment (phadej) [inline README.md]: - Disable `NSLog` for the release builds

File: README.md [hunk 2]
```diff
@@ -467,6 +467,25 @@ First you should make your app send crash logs onto a server somewhere so that y
...
+* Use [obfuscation technique](https://github.com/UrbanApps/UAObfuscatedString) for runtime protection
```
Comment (phadej) [inline README.md]: I wouldn't ever mention *security thru obscurity*, that's lazy and lame.

File: README.md (hunk 1 above)
Comment (shams-ahmed) [inline README.md]: updated

File: README.md (hunk 2 above)
Comment (shams-ahmed) [inline README.md]: it's not the best solution but it's there for a reason... i wrote this with a open-mind, storing sensitive data in keychain would be better and secure. anyways i'll remove this now.

File: README.md [hunk 3]
```diff
@@ -467,6 +467,24 @@ First you should make your app send crash logs onto a server somewhere so that y
 
 Once you have this set up, ensure that you _save the Xcode archive (`.xcarchive`)_ of every build you release. The archive contains the built app binary and the debug symbols (`dSYM`) which you will need to symbolicate crash reports from that particular version of your app.
 
+### Security
+Sensitive data such as username/password, personal user details, OAuth token etc should always be kept securely in a private location where it not accessible outside of the application. See [Apple security documentation](https://www.apple.com/business/docs/iOS_Security_Guide.pdf) for full brief on how iOS handle this area. 
+
+##### Do's 
+* Store sensitive data in Keychain. if your uncomfortable implementing Keychain library use a wrapper like [UICKeyChainStore](https://github.com/kishikawakatsumi/UICKeyChainStore)
+* Treat untrusted files and data with care, always use NSFileProtectionComplete
+* Remove personal data from memory when it not needed (i.e. self.password = nil;)
```
Comment (richeterre) [inline README.md]: Why is this needed, other than to prevent accidental logging (which is already mentioned in this list)?

Comment (shams-ahmed) [inline README.md]: prevents any logging and to remove previous state if the user goes back to the password section. i.e NSKeyedUnarchiver or user going from background to foreground etc...

**Comment by richeterre** [comment]
Thank you for contributing! I'll make some adjustments to the wording to match the overall style of the document.
